src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -32,7 +32,7 @@
     <ProjectCapability Include="UseFileGlobs"/>
     <ProjectCapability Include="EditAndContinue"/>
     <ProjectCapability Include="OpenProjectFile"/>
-    <ProjectCapability Include="GenerateNuGetPackage"/>
+    <ProjectCapability Include="Pack"/>
```

Comment:
Filed https://github.com/NuGet/Home/issues/4146

I'll remove this capability from Microsoft.Managed.DesignTime.targets